e2e/app/xpath-options.ts

```diff
@@ -0,0 +1,20 @@
+export interface XPathOptions {
+  type?: ElementType;
+  name?: string;
+  containsText?: string;
+  normalizeSpace?: string;
+  ancestorLevel?: number;
+  iconShape?: string;
+}
+
+export enum ElementType {
+  Button = 'button',
```

Comment:
enums are typically ALL_CAPS unless there's a typescript standard that says otherwise.